Patches/Rabbie The Moonrabbit/ThingDefs_Misc/Weapons_Guns.xml

```diff
@@ -76,6 +76,124 @@
 				</value>
 			</li>
 			
+			<!-- Mortar Changes added by mod require the following -->
+			
+			<li Class="PatchOperationAdd">
+				<xpath>Defs/ThingDef[defName="Shell_HighExplosive"]/statBases</xpath>
+				<value>
+					<SightsEfficiency>0.45</SightsEfficiency>
+					<ShotSpread>1.5</ShotSpread>
+					<SwayFactor>2.5</SwayFactor>
+				</value>
+			</li>
+			
+			<li Class="PatchOperationReplace">
+				<xpath>Defs/ThingDef[defName="Shell_HighExplosive"]/tools</xpath>
+				<value>
+				  <tools>
+					<li Class="CombatExtended.ToolCE">
+					  <label>Shell</label>
+					  <capacities>
+						<li>Blunt</li>
+					  </capacities>
+					  <power>2</power>
+					  <cooldownTime>1.54</cooldownTime>
+					  <chanceFactor>1.5</chanceFactor>
+					  <armorPenetrationBlunt>0.555</armorPenetrationBlunt>
+					</li>
+					<li Class="CombatExtended.ToolCE">
+					  <label>neck</label>
+					  <capacities>
+						<li>Poke</li>
+					  </capacities>
+					  <power>2</power>
+					  <cooldownTime>1.54</cooldownTime>
+					  <armorPenetrationBlunt>0.555</armorPenetrationBlunt>
+					</li>
+				  </tools>
+				</value>
+			</li>
+			
+			<li Class="PatchOperationReplace">
+				<xpath>Defs/ThingDef[defName="Shell_HighExplosive"]/verbs/li[verbClass="Verb_ShootOneUse"]</xpath>
+				<value>
+				<li Class="CombatExtended.VerbPropertiesCE">
+					<verbClass>CombatExtended.Verb_ShootCEOneUse</verbClass>
+					<hasStandardCommand>true</hasStandardCommand>
+					<defaultProjectile>Throw_81mmMortarShell_HE</defaultProjectile>
+					<warmupTime>2.5</warmupTime>
+					<range>7</range>
+					<ignorePartialLoSBlocker>true</ignorePartialLoSBlocker>
+					<targetParams>
+						<canTargetLocations>true</canTargetLocations>
+					</targetParams>
+				</li>
+				</value>
+			</li>
+			
+			
+			<li Class="PatchOperationAdd">
+				<xpath>Defs/RecipeDef[defName="MakeShell_HighExplosive"]</xpath>
+				<value>
+					<recipeUsers>
+						<li>TableMachining</li>
+					</recipeUsers>
+				</value>
+			</li>
+			
+			<li Class="PatchOperationAdd">
+				<xpath>Defs</xpath>
+				<value>
+				  <ThingDef ParentName="Base81mmMortarShell">
+					<defName>Throw_81mmMortarShell_HE</defName>
+					<label>81mm mortar shell (HE)</label>
+					<graphicData>
+					  <texPath>Things/Projectile/Mortar/HE</texPath>
```

Comment:
When wielding the shell, it's pointed backwards, which looks a bit funny. Add a rotation, perhaps?

Reply:
Actually rotated from the base mod, but I've adjusted that rotation so it's held forwards facing now.